src/bilder/images/dagster/files/.env.tmpl

```diff
@@ -0,0 +1,16 @@
+{{- $env := ( key "edx-pipeline/shared/env-suffix" ) -}}
+{{- if eq $env "qa" -}}
+{{- with secret "postgres-dagster-data-qa/creds/app" -}}
```

Comment:
I think the env name in the mount path is a relic of a single Vault cluster which we should probably remedy rather than adding complexity to the configuration templates and weaving it throughout the image.

Reply:
That is fine, but it should be something methodical I think. This is probably the time to do it?

Reply:
Yes, we should do it now before we add more dependencies on this service.